schema/src/main/jade-tables/alignment_file.table.json

```diff
@@ -33,16 +28,15 @@
     },
     {
       "name": "max_audit_flag",
-      "datatype": "string",
-      "type": "required"
+      "datatype": "string"
     },
     {
       "name": "award",
       "datatype": "string",
       "type": "required"
     },
     {
-      "name": "file_format_type",
```

Comment:
file_format_type only has values for a subset of data types, all of which get categorized as "other"